src/unity/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -1046,6 +1071,9 @@ class cannot be returned and are ignored under this format.
         >>> print('mAP: {:.1%}'.format(results['mean_average_precision']))
         mAP: 43.2%
         """
+        if iou_threshold is None: iou_threshold = self.non_maximum_suppression_threshold
+        if confidence_threshold is None: confidence_threshold = 0.25
```

Comment:
Before this change, evaluation used a confidence threshold of 0.001, but now it defaults to 0.25. Any reason behind the change in default behavior?